code/pizzabot.js

```diff
@@ -4,4 +4,73 @@ const pepperoni = "Pepperoni Pizza"
 
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+
+// validateOrderName() which should take the orderName variable as an argument and return true or false if the pizza is on the menu or not.
+// const validateOrderName = (orderName) => {
+//   if ((orderName === hawaiian) || (orderName === vegetarian) || (orderName === pepperoni)) {
+//     return true;
+//   } else {
+//     return false;
+//   }
+// }
+
+// calculateTotalCost() which takes orderQuantity and pizzaPrice as arguments and returns the total cost for the order.
+const calculationTotalCost = (orderQuantity, pizzaPrice) => {
+  return orderQuantity * pizzaPrice;
+}
+
+// calculateCookingTime() which takes orderQuantity and returns the number of minutes it will take to finish the order.
+const calculateCookingTime = (orderQuantity) => {
+  if (orderQuantity <= 2) {
+    return 10;
+  } else if (orderQuantity > 2 && orderQuantity <= 5) {
+    return 15;
+  } else {
+    return 20;
+  }
+}
+
+const greeting = (`Hey! Happy to serve your pizza. On our menu we have ${vegetarian}, ${hawaiian} and ${pepperoni}.`);
+document.getElementById('greeting').innerHTML = greeting;
+
+const greet = (event) => {
```

Comment:
There is an arrow function convention which says that if we pass to our function only one argument, we can omit parentheses `()`, and rather write :

`const greet = event => { ... }`

than

`const greet = (event) => { ... }`

Important thing to notice - this convention should be applied ONLY when we pass one argument. If we would do this with multiple arguments, it would cause `SyntaxError` to occur